composer.json

```diff
@@ -11,7 +11,7 @@
     }
   ],
   "require": {
-    "php": "^7.1.3 | ^8.0",
+    "php": "^8.0",
```

Comment:
Would you also like to remove the PHP 7 tests here https://github.com/playox/flagception-bundle/blob/master/.github/workflows/php.yml?

Reply:
it's corrected !